### PR TITLE
Fix markdowns

### PR DESCRIPTION
**Pull request by kianmeng**
Found via this command:

    markdownlint *.md --disable MD013 MD033 MD024

See a detailed description of the rules is available at
https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,42 +1,60 @@
...
 - gex crashing on attempts to perform actions when working tree clean
 - gex crashing on encountering deleted files
 
 ## [0.1.0](https://github.com/Piturnah/gex/commits/v0.1.0) - 2022-08-05
+
 ### Added
+
 - `git status` display with diff information and current branch
 - Keyboard navigation between diffs of files and hunks
 - Diff items can be expanded or collapsed with <kbd>Tab</kbd>
```
Comment (Piturnah) [inline CHANGELOG.md:60]: Please remove the changes to `CHANGELOG.md`, I am content with its current formatting style.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+# Gex
 
-[![](https://img.shields.io/crates/v/gex)](https://crates.io/crates/gex)
-[![](https://img.shields.io/crates/d/gex)](https://crates.io/crates/gex)
-[![](https://img.shields.io/crates/l/gex)](https://crates.io/crates/gex)
-[![](https://img.shields.io/github/stars/Piturnah/gex?style=social)](https://github.com/Piturnah/gex/stargazers)
+[![crates.io](https://img.shields.io/crates/v/gex)](https://crates.io/crates/gex)
+[![download](https://img.shields.io/crates/d/gex)](https://crates.io/crates/gex)
+[![license](https://img.shields.io/crates/l/gex)](https://crates.io/crates/gex)
+[![stargazers](https://img.shields.io/github/stars/Piturnah/gex?style=social)](https://github.com/Piturnah/gex/stargazers)
```
Comment (Piturnah) [inline README.md:7]: Great :+1:

**Review by Piturnah — CHANGES_REQUESTED** [review]
Just a few notes

**Comment by Piturnah** [comment]
Thank you for making those changes, and thanks for your contribution!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Comment by Piturnah** [comment]
Released in 0.3 :tada:

**Pull request merged**